packages/wearable_rotary/tizen/src/wearable_rotary_plugin.cc

```diff
@@ -0,0 +1,85 @@
+#include "wearable_rotary_plugin.h"
+
+#include <efl-extension/wearable/circle/efl_extension_rotary.h>
+#include <flutter/encodable_value.h>
+#include <flutter/event_channel.h>
+#include <flutter/event_sink.h>
+#include <flutter/event_stream_handler_functions.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_method_codec.h>
+
+#include <memory>
+
+#include "log.h"
+
+#define ROTARY_CHANNEL_NAME "flutter.wearable_rotary.channel"
```

Comment:
Prefer constexprs, or just inline the string without declaring as a variable.

```suggestion
static constexpr char kChannelName[] = "flutter.wearable_rotary.channel"
```

https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros
https://google.github.io/styleguide/cppguide.html#Use_of_const